examples/react-example/App.jsx

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable */
-const React = require('react')
-const Uppy = require('@uppy/core')
-const Tus = require('@uppy/tus')
-const GoogleDrive = require('@uppy/google-drive')
-const { Dashboard, DashboardModal, DragDrop, ProgressBar, FileInput } = require('@uppy/react')
+import React from'react'
+import Uppy from'@uppy/core'
+import Tus from'@uppy/tus'
+import GoogleDrive from'@uppy/google-drive'
+import { Dashboard, DashboardModal, DragDrop, ProgressBar, FileInput } from'@uppy/react'
 
-module.exports = class App extends React.Component {
+export default class App extends React.Component {
```

Comment:
I think it would be best to rewrite this to hook to show `useUppy`. We frequently get issues about people forgetting `useUppy` and hooks is what people write these days.